.eslintignore

```diff
@@ -39,7 +39,7 @@ snapshots.js
 /packages/kbn-test/src/functional_test_runner/lib/config/__tests__/fixtures/
 /packages/kbn-ui-framework/dist
 /packages/kbn-flot-charts/lib
-/packages/kbn-monaco/src/painless/antlr
```

Comment:
Can this be the source of the `2MB` drop?

Reply:
No, please review https://github.com/elastic/kibana/pull/146379/files#diff-37720dcf8abc68d5211a94d91df2c70969073de66f75f1a2bcd7fa8a54c9f8bb . I've moved all workers into separate chunks. It's a reason